cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/log/LogHandlerTestRun.java

```diff
@@ -207,6 +207,49 @@ public void testAdapterLogsPrev() throws Exception {
   }
 
   @Test
+  public void testWorkflowLogs() throws Exception {
+    testLogs("testTemplate1", "workflows", "testWorkflow1", MockLogReader.TEST_NAMESPACE, "testAdapter1");
+    try {
+      // in default namespace, we did not produce logs through adapters
+      testNext("testTemplate1", "workflows", "testWorkflow1", true, Constants.DEFAULT_NAMESPACE, "testAdapter1");
+      Assert.fail();
+    } catch (AssertionError e) {
```

Comment:
Here we call Assert.fail() to throw an AssertionError and then we catch it and do nothing. So what is the test trying to test? Same comment for this pattern used below.